src/Modifier/Common/AttributeModifier.php

```diff
@@ -20,6 +22,8 @@
  */
 class AttributeModifier extends Modifier
 {
+    use SymfonyContainerTrait;
+
```

Comment:
This mustn't be used here. It is only for overwritten OXID classes where dependency injection is not possible.

```suggestion
```